src/plugins/intel_gpu/src/graph/include/convolution_inst.h

```diff
@@ -127,6 +127,23 @@ class typed_primitive_inst<convolution> : public typed_primitive_inst_base<convo
     static layout calc_output_layout(convolution_node const& node, kernel_impl_params const& impl_param);
     static std::string to_string(convolution_node const& node);
 
+    bool need_reset_input_memory() const override {
+        auto input_layout = _deps[0].first->_impl_params->get_output_layout(0);
+        if (input_layout.data_padding.lower_size() != tensor(0) || input_layout.data_padding.upper_size() != tensor(0)) {
```

Comment:
We have `operator bool()` defined for data padding, so this can be simplified

Reply:
Applied